### ISSUE TITLE
No reminders sent for group events

### ISSUE THREAD

**Issue by renehamburger** [isuru89/moodle-reminders-for-calendar-events#22]
I've encountered the same exception another user documented [here](https://moodle.org/mod/forum/discuss.php?d=345196):

```
\www\moodle\local\reminders\contents\group_reminder.class.php(59): get_coursemodule_from_instance('0', '0', '9')
#1 \www\moodle\local\reminders\contents\group_reminder.class.php(42): group_reminder->load_course_object()
#2 \www\moodle\local\reminders\lib.php(365): group_reminder->__construct(Object(calendar_event), Object(stdClass), 1)
...
```

After debugging the issue, I only saw now that you already fixed this in [this commit](https://github.com/isuru89/moodle-reminders-for-calendar-events/commit/9116a412a9c1e2cd9a8d8fc34fd4c75f381563d1). Would be great if this could be released.

**Comment by isuru89** [comment]
@renehamburger  Thanks for pointing out this.

I have released the fix as v1.5.1. Please check.

**Comment by renehamburger** [comment]
Looks good, @isuru89. Thanks for the quick fix!

By the way, was there a specific reason why event descriptions are not always included in the email html? It seems that they are always included in the text-only version (albeit with html tags), but were commented out for specific event types. I've changed this locally to always include non-empty descriptions.
